### PR TITLE
Move GlxInfo to Flathub

### PR DESCRIPTION
**Pull request by TingPing**

### CODE REVIEW

File: builds.json [hunk 1]
```diff
@@ -161,6 +161,16 @@
...
+        },
+        "org.freedesktop.GlxInfo/18.08": {
+            "repo": "default",
+            "git-module": "org.freedesktop.GlxInfo.git",
+            "git-branch": "18.08"
+        },
+        "org.freedesktop.GlxInfo/1.6": {
+            "repo": "default",
+            "git-module": "org.freedesktop.GlxInfo.git",
+            "git-branch": "1.6"
```
Comment (barthalion) [inline builds.json]: You don't need it these days, just push your changes to `branch/1.6` and `branch/18.08` branches.

Comment (TingPing) [reply builds.json]: I thought that was only for runtimes not apps?

Comment (barthalion) [reply builds.json]: No, it's universal.

**Comment by TingPing** [comment]
Alright `branch/` works fine, somebody can squash this on merge.

**Pull request merged**